src/object.rs

```diff
@@ -1541,6 +1541,7 @@ impl<T: ObjectType> ObjectExt for T {
         ::signal::connect_raw(
             self.as_object_ref().to_glib_none().0,
             signal_name.as_ptr() as *const _,
+            // Two level-kind of callback, which explains why the transmute is required.
```

Comment:
I don't understand the comment

Reply:
I might have been a bit too lazy writing this comment, let me improve it.

Reply:
Same for the other such comments. What's "two levels" here exactly, or "two times" (below)?